src/nupic/math/ArrayAlgo.hpp

```diff
@@ -3786,106 +3786,7 @@ namespace nupic {
       int n2 = (int)(end - start - n1);
 
 
-#if defined(NTA_PLATFORM_darwin86_disabled) || defined(NTA_PLATFORM_linux32_disabled)
```

Comment:
It isn't clear to me what the intent of this block was for. Can you comment on why you removed it?